ggml/include/ggml-cann.h

```diff
@@ -33,6 +33,9 @@ extern "C" {
  * @brief Maximum number of CANN devices supported.
  */
 #define GGML_CANN_MAX_DEVICES 16
+#define GGML_CANN_NAME "CANN"
```

Comment:
I suspect that this macro was taken from the implementation in the CUDA backend, but the reason it exists there is because the same code is used to build the ROCm and MUSA backends, so the name of the backend may change depending on the build flags, but I don't think this is necessary in the CANN backend.

Reply:
Thanks for your review, this macro is no longer exposed to llama. I have moved this macro to the cann implementation file.